plugins/tinymce4/config/config.php

```diff
@@ -1,8 +1,6 @@
 <?php
 
-$plugin_cf['tinymce4']['init']="fontawesome";
+$plugin_cf['tinymce4']['init']="fontawesome-codemirror";
```

Comment:
init should be set to "fontawesome". 
The sourceview with codemirror plugin is prepared, But setting this option throws error "CodeMirror not found..." when setting source view.